pkg/aws/aws_suite_test.go

```diff
@@ -32,15 +32,18 @@ const (
 	testNamespace = "test"
 )
 
+//params is used to pass annotations to the machine spec
 func newMachine(
-	setMachineIndex int,
+	setMachineIndex int, params ...map[string]string,
```

Comment:
Why variadic? I see only `params[0]` being used [below](https://github.com/gardener/machine-controller-manager-provider-aws/pull/32/files#diff-22e0e177995875730b9c602d1be2aefcdd624ddec10f8be8b5e31f606bc20925R71).

Why not just `	setMachineIndex int, annotations map[string]string,`?

Reply:
Yes @amshuman-kr  I had thought of soln you are proposing earlier as well, but I have used variadic because otherwise I would have had to make changes in `newMachine()` call everywhere with second parameter as nill, while using variadic doesn't require that.
And I thought it was better for `newMachine()` be the only helper function to create new machines , and not make a new helper function just for making new machine with annotations.